src/open_prime_rando/echoes/asset_ids/__init__.py

```diff
@@ -1,5 +1,9 @@
-from . import agon_wastes, great_temple, sanctuary_fortress, temple_grounds, torvus_bog, world
+from . import agon_wastes, great_temple, sanctuary_fortress, temple_grounds, torvus_bog
 
 __all__ = [
-    "agon_wastes", "great_temple", "sanctuary_fortress", "temple_grounds", "torvus_bog", "world",
```

Comment:
why remove world?

Reply:
I made this file generated and I did an oopsie